arc/database/client.py

```
@@ -609,9 +609,18 @@ async def record_outcome(self, outcome_data: Dict[str, Any]) -> str:
         outcome_id = str(uuid.uuid4())
         
-        # Validate trajectory data
+        # Ensure trajectory meets database constraint requirements
+        trajectory = outcome_data["trajectory"]
+        if isinstance(trajectory, dict):
+            # Add required fields if missing
+            if "start_time" not in trajectory:
+                trajectory["start_time"] = outcome_data.get("execution_time", datetime.utcnow()).isoformat()
+            if "status" not in trajectory:
+                trajectory["status"] = outcome_data["status"]
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)